src/pivotal-ui/components/dropdowns/dropdowns.scss

```diff
@@ -381,6 +397,18 @@ This is a notification dropdown with alerts.
   }
 }
 
+.dropdown-toggle {
+  .caret {
+    display: inline-block;
+    border: 0;
+    margin-right: -0.6em;
```

Comment:
is this always going to match the font correctly? Did it look right with a small button (that has smaller font)?